doc/main/reference/try_put_and_wait.rst

```diff
@@ -0,0 +1,322 @@
+.. _try_put_and_wait:
+
+Waiting for Single Messages in Flow Graph
+=========================================
+
+.. contents::
+    :local:
+    :depth: 1
+
+Description
+***********
+
+This feature adds a new ``try_put_and_wait`` interface to the receiving nodes in the Flow Graph.
+This function allows to wait until completion of single message, submitted to ``try_put_and_wait`` as an input by the Flow Graph in
```

Comment:
I'd suggest, "This function puts a message as an input into a Flow Graph and then waits until all work related to that input is complete."

Reply:
I'd add a second sentence, "try_put_and_wait may reduce latency compared to calling graph::wait_for_all since graph::wait_for_all waits for all work, including work unrelated to the input message, to complete."

Reply:
Fixed